official/resnet/resnet_run_loop.py

```diff
@@ -374,6 +379,16 @@ def input_fn_eval():
     eval_results = classifier.evaluate(input_fn=input_fn_eval,
                                        steps=flags.max_train_steps)
     print(eval_results)
+    if benchmark_logger is not None:
+      # The eval_results result contains 3 entries:
+      # 1. loss, which is default output for Evaluator.evaluate
+      # 2. global_step, which is default output for Evaluator.evaluate
+      # 3. accuracy, which is defined in eval_metric_ops
+      global_step = eval_results[tf.GraphKeys.GLOBAL_STEP]
+      for key in eval_results:
```

Comment:
I feel like this should be a function of the logger if every classifier is going to return global step-- a wrapper around log_metric that says, log_dict_with_global_step or something.